test/test_MVU.sh

```diff
@@ -63,46 +62,46 @@ banner() {
 }
 
 run_make() (
```

Comment:
If this was changed to not fork (change last `(` to `}`, and the closing one on line 67) then maybe that'd fix the issue of trying to run as root. We'd need to undo the `cd` if we did that though (which is why I wrote this as a forking function in the first place).

Reply:
Removed in 4479938